CHANGES.md

```diff
@@ -30,8 +30,7 @@ Release Notes.
 * Fix deadlock problem when using elasticsearch-client-7.0.0.
 * Fix storage-jdbc isExists not set dbname.
 * Fix `searchService` bug in the InfluxDB storage implementation.
-* Fix CVE in the alarm module, when activating the dynamic configuration feature.
-* Fix CVE in the endpoint grouping, when activating the dynamic configuration feature.
+* Fix CVE of SnakeYAML when activating the dynamic configuration feature.
```

Comment:
I merge them into one, they are essential one type of fix IMO

Reply:
CVE needs to be explicit. Even we don't report the CVE-ID(s), but basically, these CVEs are different, just the same in the code style.